### ISSUE TITLE
Confusing reference to a doc that doesn't exist in newer versions

### ISSUE THREAD

**Issue by zarantha2003** [NetAppDocs/active-iq-unified-manager#13]
Page: [Getting started with Active IQ Unified Manager](https://docs.netapp.com/us-en/active-iq-unified-manager/api-automation/concept_get_started_with_um_apis.html)

"The Unified Manager API Developer’s Guide" -> The page referencing the Developer's Guide is under the "Manage storage using REST APIs" section. That Manage storage section is what has replaced the Developer's guide PDF.

We should probably change "The Unified Manager API Developer’s Guide" to the section title or otherwise reword it to be less confusing. As it stands, we have customers looking for the Developer's Guide and finding old links.

9.7 link to compare sections with the sub sections under "Manage storage using REST APIs" for confirmation:
http://docs.netapp.com/ocum-97/topic/com.netapp.doc.onc-um-api-dev/Managing%20your%20environment%20using%20REST%20APIs.pdf

**Comment by zarantha2003** [comment]
Or we can change the name of the top section "Manage storage using REST APIs" back to the Developer's guide. Either works as long as we're consistent.

We have multiple links in the doc that link back to this section that are titled Developer's Guide:
https://docs.netapp.com/us-en/active-iq-unified-manager/storage-mgmt/concept_use_unified_manager_rest_apis_ocum.html
https://docs.netapp.com/us-en/active-iq-unified-manager/config/concept_policy_based_storage_management.html
https://docs.netapp.com/us-en/active-iq-unified-manager/reporting/generaltask_um_rest_apis_that_can_be_used_for_reporting.html
https://docs.netapp.com/us-en/active-iq-unified-manager/config/task_perform_initial_setup_of_unified_manager_web_ui.html
https://docs.netapp.com/us-en/active-iq-unified-manager/health-checker/concept_common_unified_manager_administrative_workflows_and_tasks.html

Another page where the Developer's Guide is referenced but not linked:
https://docs.netapp.com/us-en/active-iq-unified-manager/config/concept_api_gateway.html

**Comment by netapp-manini** [comment]
Thank you for your feedback. We will update the content to make sure it aligns with what you have suggested. We will update this issue with the links to the updated content and inform you.

**Comment by netapp-manini** [comment]
Hi @zarantha2003 

The generic references to guides have been replaced with appropriate text. Please check the links you added in the issue.
